Templates/Templates/AutoGenerateProtocol.stencil

```diff
@@ -0,0 +1,32 @@
+
+//: Do not change this code as it is autogenerated every time you build.
+//: You can change the code in `../StencilTemplatesForSourcery/Application/AutoGenerateProtocol
+
+// MARK: - AutoGenerateProtocol
+//: From all Types implementing this protocol Sourcery adds:
```

Comment:
Not sure if documentation comments are needed in the generated code. I've proposed a format for templates documentation here #531 but didn't touch on that. @krzysztofzablocki what do you think?

Reply:
Might be useful for people unfamiliar with Sourcery, but if we do add docs to generated code then I'd be 👍 for using the format you described, that we could eventually actually parse and run some extra checks on the tool side